docker/build-rpm-in-docker.sh

```diff
@@ -1,6 +1,48 @@
 #!/bin/sh
-cd $(dirname $0)
-mkdir -p build-dir-$1
-cp ../* build-dir-$1
-cp -r ../dist build-dir-$1/dist
-sudo docker run -i -t -v ${PWD}/build-dir-$1:/build/build-dir-$1 erlang-rpm-build-$1
+#
+#
+#
+centos_version="$1"
+build_dir="build-dir-$centos_version"
+packages=( "Makefile" "*.patch" "erlang.spec" "Erlang_ASL2_LICENSE.txt" )
```

Comment:
This kind of array declaration works only using `bash`. 

`sh` does not support declaration in this way.

`sh` does not work:
```
sh  build-rpm-in-docker.sh 6
build-rpm-in-docker.sh: 7: build-rpm-in-docker.sh: Syntax error: "(" unexpected
```

`bash` works:
```
bash  build-rpm-in-docker.sh 6
rm -rf BUILDROOT BUILD SOURCES SPECS SRPMS RPMS tmp FINAL_RPMS dist
```

as you can see: ` #!/bin/sh` we are using `sh`. 

you should modify it and find another way.